packages/grafana-ui/src/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,82 @@
+import { css } from '@emotion/css';
+import { FocusScope } from '@react-aria/focus';
+import React, { useState } from 'react';
+import { usePopperTooltip } from 'react-popper-tooltip';
+import { CSSTransition } from 'react-transition-group';
+
+import { ReactUtils } from '../../utils';
+import { Portal } from '../Portal/Portal';
+import { TooltipPlacement } from '../Tooltip/types';
+
+export interface Props {
+  overlay: React.ReactElement | (() => React.ReactElement);
+  placement?: TooltipPlacement;
+  children: React.ReactElement;
+  /** Defaults to click */
+  trigger?: Array<'click' | 'hover'>;
+}
+
+export const Dropdown = React.memo(({ children, overlay, placement, trigger = ['click'] }: Props) => {
+  const [show, setShow] = useState(false);
+  const { getArrowProps, getTooltipProps, setTooltipRef, setTriggerRef, visible } = usePopperTooltip({
+    visible: show,
+    placement: placement,
+    onVisibleChange: setShow,
+    interactive: true,
+    delayHide: 200,
+    delayShow: 150,
+    offset: [0, 8],
+    trigger: trigger,
+  });
+
+  const animationDuration = 150;
```

Comment:
**Question:** is this the same as what is set for `delayShow: 150`? If yes maybe we could move it to the top and use it in `usePopperTooltip()` as well.

Reply:
hm.. think I need to remove the delayShow that should not be there  (should be 0)